src/pages/admin/routes.tsx

```diff
@@ -14,7 +14,7 @@ export const AdminRoutes = (
     <div>
       <Route path={adminRouteEnums.default} component={DashboardPage}/>
       <Route path={adminRouteEnums.student.list} component={ListStudentPageContainer}/>
-      <Route path={adminRouteEnums.student.edit} component={EditStudentPage}/>
+      <Route path={`${adminRouteEnums.student.edit}/:id`} component={EditStudentPageContainer}/>
```

Comment:
Try to avoid "magic" strings. Take a look how is implement for example in 
#### ./src/common/routeEnums/trainer/index.ts